tests/Properties.hs

```diff
@@ -1614,6 +1614,36 @@ prop_read_write_file_D x = unsafePerformIO $ do
         (const $ do y <- D.readFile f
                     return (x==y))
 
+prop_hgetline_like_s8_hgetline (LinedASCII filetext) (lineEndIn, lineEndOut) = idempotentIOProperty $ do
+    tid <- myThreadId
+    let f = "qc-test-"++show tid
+    let newlineMode = NewlineMode (if lineEndIn then LF else CRLF) (if lineEndOut then LF else CRLF)
+    bracket_
```

Comment:
Not much point to use `bracket_` in tests. In this particular case it even makes things worse, because if the test fails with an exception, I would rather prefer do not `removeFile f` to be able to investigate.